cmd/payload-testing/prow-plugin/server.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/promotion"
+	configresolver "github.com/openshift/ci-tools/pkg/registry/server"
+	"github.com/sirupsen/logrus"
+
+	prowconfig "k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+)
+
+type githubClient interface {
+	CreateComment(owner, repo string, number int, comment string) error
+}
+
+var ocpPayloadTestsPattern = regexp.MustCompile(`(?mi)^/ocp-payload-tests\s*$`)
+
+func helpProvider(_ []prowconfig.OrgRepo) (*pluginhelp.PluginHelp, error) {
+	// TODO(muller): Better descriptions, better help
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The ocp-payload-testing plugin spawns a battery of OCP release qualification testsuite jobs`,
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		// TODO(muller): What commands do we actually need?
+		// We will likely want to specify which jobs: blocking | informing | analysis | periodics | all
+		// We will also likely want to specify which release: ci | nightly | arm64 | s390x | ppc64
```

Comment:
We can't build those for PRs so not yet

Reply:
> We can't build those for PRs so not yet

ci vs nightly (in this list) is important. The set of jobs run on nightlys is different than the set of jobs for ci

Reply:
Yeah, but it somehow needs to be systematic: from the outside `arm64` is no different than`nightly`, but we actually need to treat them differntly (because we can do our thing with `nishglty `but not with `arm64`)